packages/hosted-credit-card-integration/src/HostedCreditCardPaymentMethod.tsx

```diff
@@ -276,5 +276,6 @@ export default toResolvableComponent<PaymentMethodProps, PaymentMethodResolveId>
             id: 'hosted-credit-card',
             // type: 'PAYMENT_TYPE_API',
```

Comment:
Can I ask why is this bit commented out as well? Also these are supposed to be helper components. What I mean by that is that `BluesnapComponent` package will import and will be composed using this.

Reply:
> Can I ask why is this bit commented out?

**A:** It's not my change but I found the answer 👉 https://github.com/bigcommerce/checkout-js/pull/1146#discussion_r1065220457

> Also these are supposed to be helper components.

**Q:** Do you mean that I am implementing it wrong? Could you give me an example of how to use it correctly? So, what's the purpose of being a resolvable component? I thought it was intended to work the same way as new payment strategies.

Reply:
Hi @mauricio-sg , Animesh and I talked about it, and it came to our attention that you edited this package.

Ideally, with the helper CC component(https://github.com/bigcommerce/checkout-js/tree/master/packages/credit-card-integration), you should be able to build your own credit card payment method component similar to the `hosted-credit-card-integration`. You don't need to touch this file.

Furthermore, as a new payment package, we highly suggest you to have an e2e test to ensure that no future modifications could break it.

Resources:
- e2e test code sample: https://github.com/bigcommerce/checkout-js/blob/master/packages/barclay-integration/e2e/barclay.spec.ts
- e2e testing Doc: https://bc-eng-docs.herokuapp.com/domains/checkout/payment-integrations-contributing-guide-checkout.html#e2e-test

Reply:
Going back to this if this is still commented, can we remove this?

Reply:
@mauricio-sg